WordPress/Classes/ViewRelated/NUX/SiteInfoHeaderView.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+/// A subclass of BlogDetailHeaderView styled for use in the login flow.
+///
+class SiteInfoHeaderView: BlogDetailHeaderView {
+
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        configureStyles()
+    }
+
+
+    func configureStyles() {
+
+        titleLabel.font = WPFontManager.systemSemiBoldFont(ofSize: 15.0)
```

Comment:
Specifying the font size directly will make it non-dynamic. I think the call you want is `WPStyleGuide.fontForTextStyle(.subhead, fontWeight: .semiBold)`. Subhead should be 15pt, but you'll want to double check this call works otherwise ;)